scripts/SHI/scpSHI_Jetz_complete.yml

```diff
@@ -0,0 +1,93 @@
+script: scpSHI_Jetz_complete.R
+description: 
+  This script loads the data needed to calculate the area and connectivity scores 
+  to calculate the Species Habitat Index (SHI) using Global Forest Watch data to measure 
+  changes in the habitat of the species
+inputs:
+  species:
+    label: species
+    description: Scientific name of the species
+    type: text
+    example: Saguinus oedipus
+  ref_system:
+    label: Projection
+    description: Reference system for the area of interest
+    type: int
```

Comment:
Wu used for other scripts type: text
User can use (see https://spatialreference.org/ref/epsg/3116/):
ESPG:3116 or
+proj=tmerc +lat_0=4.596200416666666 +lon_0=-74.07750791666666 +k=1 +x_0=1000000 +y_0=1000000 +ellps=GRS80 +towgs84=0,0,0,0,0,0,0 +units=m +no_defs

Reply:
but isn't it easier to write just the number and less prone to error?

Reply:
Replaced with:

 srs:
    label: spatial reference system
    description: Reference system for the area of interest. It can be 1) the Spatial Reference System Identifier (SRID), 2) the authority name (e.g. EPSG) with the SRID or 3) the description of the spatial reference system details (e.g. [https://spatialreference.org/]). If just the SRID is given and the software can not find the reference system try options 2 or 3.
    type: text
    example: EPSG:3116

And for the R code in order to use the 3 options I added this: 

```
#Define SRS
srs <- input$srs
check_crs <- grepl("^[[:digit:]]+$",srs)
sf_crs  <-  if(check_crs) st_crs(as.numeric(srs)) else  st_crs(srs) # converts to numeric in case SRID is used
srs_cube <- if(check_crs){
  authorities <- c("EPSG","ESRI","IAU2000","SR-ORG")
  auth_srid <- paste(authorities,srs,sep=":")
  auth_srid_test <- map_lgl(auth_srid, ~ !"try-error" %in% class(try(st_crs(.x))))
  if(sum(auth_srid_test)>1) print("--- Please specify authority name or provide description of the SRS  ---") else auth_srid[auth_srid_test] 
}else  srs # paste Authority in case SRID is used 
```